documentation/modules/post/multi/gather/irssi_creds.md

```diff
@@ -0,0 +1,27 @@
+## Vulnerable Application
+
+  Any system with a `shell` or `meterpreter` session.
+
+## Verification Steps
+
+  1. Get a `shell` or `meterpreter` session on some host.
+  2. Do: ```use post/multi/gather/irssi_creds```
+  3. Do: ```set SESSION [SESSION_ID]```, replacing ```[SESSION_ID]``` with the session number you wish to run this one.
+  4. Do: ```run```
+  5. If the system has readable configuration files containing irc passwords, they will be printed out.
```

Comment:
Yes, they are irc passwords coming from irssi.  I decided to use IRC instead because that's the interesting bit.  If there's a preference to use irssi in the comments, I'm happy to do it if needed.